gui/setup.iss

```diff
@@ -25,6 +23,7 @@ DefaultDirName={pf}\GPSBabel
 DefaultGroupName=GPSBabel
 OutputDir=release
 OutputBaseFilename=GPSBabel-1.5.4-Setup
+OutputManifestFile=GPSBabel-1.5.4-Manifest.txt
 SetupIconFile=images\babel2.ico

```

Comment:
Wow, there's an icon file from the (pre innosetup?)  past. Must find and destroy... and add "qtinstaller" to my list of things to learn more about.